process/smartContract/scQueryService.go

```diff
@@ -79,7 +79,7 @@ func (service *SCQueryService) createVMCallInput(query *process.SCQuery, gasPric
 		CallerAddr:  query.ScAddress,
 		CallValue:   big.NewInt(0),
 		GasPrice:    gasPrice,
-		GasProvided: service.economicsFee.MaxGasLimitPerBlock(),
+		GasProvided: service.economicsFee.MaxGasLimitPerBlock(0),
```

Comment:
It is ok to be harded here to shard 0 ?

Reply:
it is a query service it does not matter.